dist/index.js

```diff
@@ -3,11 +3,10 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-const requireDir = require('require-dir');
-const rules = requireDir("./rules");
-const configs = requireDir("./configs");
+const rules = require("./rules/index.js");
+const configs = require("./configs/index.js");
 
 module.exports = {
-  rules,
-  configs
+  ...configs,
```

Comment:
do these needs to be spread?

Reply:
the setup that i have found to work is to leave the spread operator on the rules object in the main index.js file due to the way that the index.js file in the rules subfolder needs to be setup properly use the rules in the configurations.

Reply:
due to the nested nature of barrel files rules now has a spread operator and configs do not. this equates to the final export having a rules and configs object that are not nested